tools/src/dbgen/Lib.hs

```diff
@@ -97,6 +97,9 @@ data AccountSpec = AccountSpec
 instance FromJSON AccountSpec
 instance ToJSON AccountSpec
 
+type AddressRange = Integer
+type DistributionAmount = Integer
```

Comment:
I'd be happier to see a `newtype` here, as they're used in `RangeDistribution` side-by-side. It would be easy to forget which was which.

Reply:
Good catch @parsonsmatt ! Missed that!

Reply:
@parsonsmatt: I've made these changes. When you get a chance, let me know what you think.